Hudl.Mjolnir.Tests/Bulkhead/SemaphoreBulkheadHolderTests.cs

```diff
@@ -85,31 +83,31 @@ public void Construct_SetsInitialBulkhead()
             Assert.Equal(expectedMaxConcurrent, holder.Bulkhead.CountAvailable);
         }
 
-        [Fact]
-        public void Construct_CallsAddChangeHandler()
-        {
-            // Arrange
-
-            var key = AnyGroupKey;
-            var mockMetricEvents = new Mock<IMetricEvents>(MockBehavior.Strict);
-            mockMetricEvents.Setup(m => m.BulkheadGauge(It.IsAny<string>(), It.IsAny<string>(), It.IsAny<int>(), It.IsAny<int>()));
-
-            var mockBulkheadConfig = new Mock<IBulkheadConfig>(MockBehavior.Strict);
-            mockBulkheadConfig.Setup(m => m.GetMaxConcurrent(key)).Returns(AnyPositiveInt);
-            mockBulkheadConfig.Setup(m => m.AddChangeHandler(key, It.IsAny<Action<int>>()));
-
-            var mockLogFactory = new Mock<IMjolnirLogFactory>(MockBehavior.Strict);
-            mockLogFactory.Setup(m => m.CreateLog(It.IsAny<Type>())).Returns(new DefaultMjolnirLog());
-
-            // Act
-
-            var holder = new SemaphoreBulkheadHolder(key, mockMetricEvents.Object, mockBulkheadConfig.Object, mockLogFactory.Object);
-
-            // Assert
-
-            // It.IsAny<Action<int>>() could be better here, but I can't access the non-static method group.
-            mockBulkheadConfig.Verify(m => m.AddChangeHandler(key, It.IsAny<Action<int>>()), Times.Once);
-        }
+//        [Fact]
```

Comment:
Is this getting re-implemented? Or should it be deleted?

Reply:
This has been done in https://github.com/hudl/Mjolnir/pull/64